docs/DeveloperGuide.md

```diff
@@ -324,6 +324,11 @@ In this section, we will outline the key operations of the Module Manager, namel
 * `AddModuleCommand`
 * `DeleteModuleCommand`
 * `EditModuleCommand`
+
+Before we begin, it is important for us to define 1 key term: **association**:
+
+:information_source: A lesson/task/contact is associated with a module if its **code** field is equal to the
+ module's **code**.
```

Comment:
Actually "a contact is associated with a module if it contains that module'**code** as one of its tags"